app/api-fetch/album-by-id.ts

```
@@ -39,6 +39,14 @@
 
 export default async function fetchAlbumById(id: string): Promise<AlbumDetails> {
   try {
+    if (localStorage && localStorage.getItem(
+      `playlist-${id}`
+    )) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)